src/cider/nrepl/middleware/util/nrepl.clj

```diff
@@ -19,3 +19,41 @@
                              :status :notification
                              :msg notification
                              (when type [:type type])))))
+
+(defn- update-some
```

Comment:
I think you meant to put all of those in some of the other util namespaces. Those here are special middleware utils that result in sending messages to the clients.

Reply:
I've added a util.coerce namespace for this. The `transform-value` stuff got moved into orchard, and I think it should be moved back eventually, perhaps even for this purpose.

Reply:
Yeah, probably. I think it doesn't make much sense outside its usage with nREPL.